build_n_deploy/webpack/webpack.prod.js

```diff
@@ -23,6 +24,14 @@ const config = mergeWithCustomize({
     devtool: 'source-map',
     module: {
         rules: [
+            {
+                test: /\.(jsx|tsx|ts|js)?$/,
+                exclude: /node_modules/,
```

Comment:
🔥🔥🔥

Reply:
Denne lå i common, men vi har en egen i dev for "refresh" - som gjør at vi får "dobbelt" i dev hvis denne ligger i common. Usikker på om det spiller noen rolle, men uryddig uansett.